presto-main/src/test/java/io/prestosql/operator/TestTopNRowNumberOperator.java

```diff
@@ -193,7 +193,7 @@ public void testUnPartitioned(boolean partial)
     public void testMemoryReservationYield()
     {
         Type type = BIGINT;
-        List<Page> input = createPagesWithDistinctHashKeys(type, 6_000, 600);
+        List<Page> input = createPagesWithDistinctHashKeys(type, 1_000, 500);
```

Comment:
Should this decrease `1_000_000` later in the test body:?

Reply:
That value is `additionalMemoryInBytes` which I don't think is related.